README.md

```diff
@@ -43,26 +44,54 @@ To test new features, you can use ydb version that is currently in testing mode
 or use a build from `main` brunch with `ghcr.io/ydb-platform/local-ydb:trunk` image.
 Also you can set specific version like `cr.yandex/yc/yandex-docker-local-ydb:23.1`
 
-### Custom backend in dev mode
+### Custom configuration in dev mode with .env file
 
-YDB docker represents a single node cluster with only one version, small amount of storage groups, PDisks and VDisks. It may be not enough for development purposes. If you have your own development cluster with sufficient amount of entities, you can run the app in the dev mode with this cluster as backend. To do it, set you host to `REACT_APP_BACKEND` variable in `dev` script. For example:
+You can run the app with your own params by adding `.env` file to project root.
+
+1. Add `.env` file to project root.
+
+```
+DISABLE_ESLINT_PLUGIN=true
+TSC_COMPILE_ON_ERROR=true
+REACT_APP_BACKEND=http://localhost:8765
+REACT_APP_META_BACKEND=undefined
+META_YDB_BACKEND=undefined
+```
+
+2. Run `npm run start`. Your custom params from `.env` file will be applied
+
+#### Custom backend in dev mode
+
+YDB docker represents a single node cluster with only one version, small amount of storage groups, PDisks and VDisks. It may be not enough for development purposes. If you have your own development cluster with sufficient amount of entities, you can run the app in the dev mode with this cluster as backend. To do it, alter `REACT_APP_BACKEND` param in your `.env` file:
 
 ```
-"dev": "DISABLE_ESLINT_PLUGIN=true TSC_COMPILE_ON_ERROR=true REACT_APP_BACKEND=http://your-cluster-host:8765 npm start"
+DISABLE_ESLINT_PLUGIN=true
```

Comment:
It seems that's no need to repeat all variables. Let's here and farther keep only changed variables here?